### DISCUSSION TITLE
Downtime for https://opnxng.com

### DISCUSSION THREAD

**Discussion by extremelyonline** [searxng/searx-instances#305] [Report instance downtime]
Hello, my instance will be down in the next 6 hours. I am updating the settings.yml and encountered some errors. I will revert back as soon as the troubleshooting is done.

**Comment by extremelyonline** [comment]
The problem has been fixed now, to give a report:

I was using the Paulgo fork and noticed that my settings.yml is quite outdated. I tried to modified it and to use _use_default_settings: true_.  

I was hoping that this change will take a few minutes at best. But it crashed my instance, presumably due to an incompatibility between the official settings and the Paulgo fork.

Anyway, I was able to retrieve the [latest settings.yml of the Paulgo fork](https://github.com/paulgoio/searxng/issues/16#issuecomment-1138606614) and get the instance working again. I am sorry for the downtime and will be more careful in the future.

**Comment by unixfox** [comment]
Thank you for the notification.

**Comment by extremelyonline** [comment]
Thank you @unixfox. It seems that the instance is not in the public list. May I request for it to be added back?

**Reply by unixfox** [reply]
Is it already in the list: https://github.com/searxng/searx-instances/blob/master/searxinstances/instances.yml#L22

**Reply by extremelyonline** [reply]
Yes. Thank you!
